tests/pkg/minmea/main.c

```diff
@@ -111,6 +111,9 @@ int _parse_nmea_msg(const char* nmea_msg)
             time_t timestamp;
             minmea_getdatetime(&tm, &frame.date, &frame.time);
             timestamp = mktime(&tm);
+            /* timestamp is dependent on given EPOCH time */
```

Comment:
```suggestion
            timestamp = mktime(&tm);
            /* minmea is providing a UTC date time (decoded from the GPS Message) 
             mktime() assumes local time, the timestamp might therefor depend on EPOCH and timezone*/
             
```

Reply:
kwark we should just use timegm()
```suggestion
            timestamp = timegm(&tm);
```

Reply:
timegm might not be supported by all libc

Reply:
or we fix native #20023

Reply:
`timegm` is not supported by the MCUs. I just remove the timestamp generation and we are just checking the raw time values here. The only function that is not checked here is `minmea_getdatetime()`.

Reply:
The `minmea_getdatetime()` does just fill the `struct tm`, so there should be no problem if we exclude that. [see](https://github.com/kosma/minmea/blob/e368d847d75abd891c651f7d30ce5efb6681adb7/minmea.c#L641)

There is also the `minmea_gettime()` this also uses `timegm()` but I think we can only test that on native. [see](https://github.com/kosma/minmea/blob/e368d847d75abd891c651f7d30ce5efb6681adb7/minmea.c#L663)